src/main/java/de/retest/recheck/ignore/SearchFilterFiles.java

```diff
@@ -26,6 +27,21 @@
 
 	private SearchFilterFiles() {}
 
+	public static List<Path> getAllFilterFiles() {
+		final List<Path> filterFiles = new ArrayList<>();
+		if ( !getDefaultFilterFiles().isEmpty() ) {
```

Comment:
What is the difference between this line and line 36? And why is 36 wrapped inside a try-catch?

Reply:
getProjectFilterFiles() using Files.walk which can cause an IOException